addons/project/models/project.py

```diff
@@ -920,7 +920,7 @@ def _read_group_personal_stage_type_ids(self, stages, domain, order):
         string='Email', readonly=False, store=True, copy=False)
     partner_phone = fields.Char(
         compute='_compute_partner_phone', inverse='_inverse_partner_phone',
-        string="Phone", readonly=False, store=True, copy=False)
+        string="Phone", readonly=False, store=True, copy=False, tracking=True)
```

Comment:
Actually there is a 'phone' field in 'res.partner' in which tracking is true. And here is a compute field so we have to add tracking here also.

Reply:
With the compute and inverse methods, this field seems to be a related field, so we duplicate the tracking, IMO.
One in `res.partner` and another for `project.task`. 🤔
By the way, since it is more a related field, I don't know why this field is stored. Do you know why this field is stored ?